### PR TITLE
feat(about): Created about page based on designs

### PR DESCRIPTION
**Pull request by Mautjee**
## Description

Updated the about page to match the new designes
 

## Changes Made

- [FE] Added about page

### CODE REVIEW

File: src/pages/about.tsx [hunk 1]
```diff
@@ -1,12 +1,49 @@
-import { Typography } from "@mui/material";
+import { Stack, Typography } from "@mui/material";
 
 import { text } from "~/assets/text/index.js";
+import { styles } from "~/styles/pages/about.js";
+
+const wordsToLink: Record<string, string> = {
+  "Kryha Labs": "https://kryha.io",
+  Aleo: "https://aleo.org",
```
Comment (ramidecodes) [inline src/pages/about.tsx:8]: Shouldn't these also be a String?

File: src/pages/about.tsx [hunk 2]
```diff
@@ -1,12 +1,49 @@
...
+  zPass: "https://zpass.aleo.org/",
+};
 
 function About() {
+  const getContent = () => {
```
Comment (ramidecodes) [inline src/pages/about.tsx:13]: You could abstract some of these utilities that map and check for link outside of the page file.

**Review by ramidecodes — APPROVED** [review]

File: src/pages/about.tsx (hunk 1 above)
Comment (Mautjee) [reply src/pages/about.tsx:8]: These are strings but prettier is removing the double quotes for the words

Comment (ramidecodes) [reply src/pages/about.tsx]: Interesting...

File: src/assets/text/about.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    `And where do we fit in? We're the BotBusters, making sure that thin line between real and artificial remains strong. We hunt down and bust rogue AI's to better protect the net.`,
+    `Enter the BotBuster Program. It's a game that feeds questions into a chatroom. The kind of questions that humans like you and I can answer, but still trip up those nasty AI.`,
+    `Stay sharp. Each conversation is a clever dance of wits and gut feelings, separating genuine human interactions from sly AI tricks. These bots will say anything to convince you they’re the real deal.`,
+    `BotBusters is built by Kryha Labs on the zero-knowledge network Aleo using real privacy-preserving technology with zPass that can prove who's a real human without revealing their secrets to anyone. We can't even see it.`,
+    `While playing, you can earn points based on how well you do. Every day, the top BotBusters players with the most points will receive Aleo Credits, and this bounty resets every 24 hours.`,
+    `Plus, if you're into creating your own zk solutions, you'll be happy to know Botbusters is open source. You're free to use and modify everything we've made for your own projects.`,
```
Comment (silimarius) [inline src/assets/text/about.ts:10]:
```suggestion
    `Plus, if you're into creating your own zk solutions, you'll be happy to know BotBusters is open source. You're free to use and modify everything we've made for your own projects.`,
```

**Review by silimarius — APPROVED** [review]

**Review by carlos-kryha — APPROVED** [review]

**Pull request merged**